go-sdk/common/format.go

```diff
@@ -47,12 +50,24 @@ const (
 	RequestsPerSecondsUnit ThroughputUnit = "requests/sec"
 	RowsPerSecondsUnit     ThroughputUnit = "rows/sec"
 	WritesPerSecondsUnit   ThroughputUnit = "writes/sec"
+	AustralianDollarUnit   CurrencyUnit   = "aud"
+	CanadianDollarUnit     CurrencyUnit   = "cad"
+	SwissFrancUnit         CurrencyUnit   = "chf"
+	RenminbiUnit           CurrencyUnit   = "cny"
+	EuroUnit               CurrencyUnit   = "eur"
+	PoundUnit              CurrencyUnit   = "gbp"
+	HongKongDollarUnit     CurrencyUnit   = "hkd"
+	IndianRupeeUniit       CurrencyUnit   = "inr"
+	YenUnit                CurrencyUnit   = "jpy"
+	SouthKoreanWonUnit     CurrencyUnit   = "krw"
+	NorwegianKroneUnit     CurrencyUnit   = "nok"
+	NewZealandDollarUnit   CurrencyUnit   = "nzd"
```

Comment:
it seems it is missing the currency `"usd" | "sek" | "sgd"`. Can you add them please ?

Reply:
and since you started to sort them in alphabetic order (at least for the first twelve), can you do the same for the `"usd" | "sek" | "sgd"`